src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/DefaultRazorProjectChangePublisherTest.cs

```diff
@@ -167,7 +167,7 @@ internal async Task ProjectManager_Changed_EnqueuesPublishAsync(ProjectChangeKin
         {
             // Arrange
             var serializationSuccessful = false;
-            var projectSnapshot = CreateProjectSnapshot("/path/to/project.csproj", new ProjectWorkspaceState(ImmutableArray<TagHelperDescriptor>.Empty, CodeAnalysis.CSharp.LanguageVersion.Default));
+            var projectSnapshot = CreateProjectSnapshot("/path/to/project.csproj", new ProjectWorkspaceState(ImmutableArray<TagHelperDescriptor>.Empty, CodeAnalysis.CSharp.LanguageVersion.CSharp7_3));
```

Comment:
Out of curiosity why does this need to be `CSharp7_3`?

Reply:
Doesn't have to be. Could have been any version actually. My intent was to build two `ProjectWorkspaceState`s that were different 😄 

I could have also added TagHelpers to one; however, I didn't want to construct all the boiler plate for TagHelpers